src/stylesheets/channels.less

```diff
@@ -29,6 +29,12 @@
     }
 
     .channel-header {
+        .channel-icon, .channel-name {
+            @media (max-height: 400px) {
```

Comment:
Does the setting header only disappear on mobile screens? If so I would also add a `(max-width: @screen-sm-min)` to the media query.